cloud/amazon/iam_policy.py

```diff
@@ -27,54 +27,34 @@
     required: true
     default: null
     choices: [ "user", "group", "role"]
-    aliases: []
   iam_name:
     description:
       - Name of IAM resource you wish to target for policy actions. In other words, the user name, group name or role name.
     required: true
-    aliases: []
   policy_name:
     description:
       - The name label for the policy to create or remove.
-    required: false
-    aliases: []
+    required: true
   policy_document:
     description:
       - The path to the properly json formatted policy file (mutually exclusive with C(policy_json))
     required: false
-    aliases: []
   policy_json:
     description:
       - A properly json formatted policy as string (mutually exclusive with C(policy_document), see https://github.com/ansible/ansible/issues/7005#issuecomment-42894813 on how to use it properly)
     required: false
-    aliases: []
   state:
     description:
       - Whether to create or delete the IAM policy.
-    required: true
-    default: null
+    required: false
```

Comment:
@wimnat, 

@jimi-c and I discussed this offline. Let's go ahead and remove the default and re-add required: true. The long term plan is to over time deprecate all `implicitly present` logic across all modules. Given that, it makes sense to stop the practice.